.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
```

Comment:
The latest version seems to be 1.0.2 - https://github.com/dotnet/core/releases
IMO we better use the closest supported version e.g. 1.0.1